pkgs/development/python-modules/trio/default.nix

```diff
@@ -37,6 +37,13 @@ buildPythonPackage rec {
     "fallback_when_no_hook_claims_it"
   ];
 
+  disabledTestPaths = [
+    # using deprecated pyopenssl features
+    # https://github.com/python-trio/trio/issues/2000
```

Comment:
Surely these can be conditionally disabled for >= 3.10?

Reply:
done